CHANGELOG.md

```diff
@@ -4,6 +4,301 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.12.0-rc1](https://github.com/go-gitea/gitea/releases/tag/v1.12.0-rc1) - 2020-05-17
+
+* BREAKING
+  * Remove migration support from versions earlier than 1.6.0 (#10026)
+  * Enable ENABLE_HARD_LINE_BREAK by default for rendering markdown (#11162)
+  * Fix sanitizer config - multiple rules (#11133)
+  * [API] When using API CreateRelease set created_unix to the tag commit time (#11218)
```

Comment:
```suggestion
  * When using API CreateRelease set created_unix to the tag commit time (#11218)
```
#11218

Reply:
I wanted to specifically tag API breaking changes